src/main/java/org/openmrs/uitestframework/test/TestData.java

```diff
@@ -575,6 +577,32 @@ public static String createPerson(PersonInfo person) {
 		person.id = getPersonId(person.uuid);
 		return person.uuid;
 	}
+	
+	/**
+	 *Create a location without tags
+	 *
+	 * @param locationName the name of the location
+	 * @return the location's uuid
+	 */
+	
+	public static String createLocation(String locationName){
+		
+		TestLocation testLocation = new TestLocation(locationName);
+		return testLocation.create();
+	}
+	/**
+	 * Delete a resource permanently
+	 * 
+	 * @param uuid of the resource
+	 */
+	
+	
+	public static void permanetDelete(String uuid){
+		
+		if(null != uuid && !uuid.isEmpty()){

```

Comment:
Ok, I'll correct it  on my PC and then commit the code again. Is it fine with you?